### DISCUSSION TITLE
Custom Theme

### DISCUSSION THREAD

**Discussion by ghost** [pterodactyl/panel#4423] [Support] [answered]
Hello, I was wondering how do I make a Custom Theme and what files should I mess with.

**Comment by docimin** [comment, marked as answer]
Hey,

Almost all files can be changed. I recommend looking through the files and understand how it's all connected.
You need to know HTML & PHP.

Read through this: https://github.com/pterodactyl/panel/blob/develop/BUILDING.md
I also recommend look at the "app" and the "public" folder. :)

If you have questions there's also the "api-and-modding" channel in Discord: https://discord.gg/pterodactyl
